sunpy/map/tests/test_mapbase.py

```diff
@@ -194,6 +194,9 @@ def test_coordinate_frame(aia171_test_map):
     assert frame.observer.lon == aia171_test_map.observer_coordinate.frame.lon
     assert frame.observer.radius == aia171_test_map.observer_coordinate.frame.radius
     assert frame.obstime == aia171_test_map.date
+    
+def test_HMI_file(genric_map):
+    assert generic_map.sunpy.data.test.resampled_hmi.fits == genric_map.carrington_longitude - sunpy.coordinates.get_sun_L0(generic_map.date)
 
```

Comment:
I strongly recommend running any test code that you write. To do so, you need to `pytest sunpy/map/tests/test_mapbase.py`, and this will run the entire file. 

What you need to do for this test is:

1. Create a `pytest.fixture`for the hmi test file much like `aia171_test_map` at the top of the file. Here is the [hmi test file to see how to access the test file](https://github.com/sunpy/sunpy/blob/cd8618e961b4140f2d65cc955f48760098e068f9/sunpy/map/sources/tests/test_hmi_source.py).

2. You will will want to assert that the `hmi_map.heliographic_longitude` is the same as if you opened the hmi file, loaded the header/meta information and done the operation on the crln_obs yourself.

Reply:
> I strongly recommend running any test code that you write. To do so, you need to `pytest sunpy/map/tests/test_mapbase.py`, and this will run the entire file.
> 
> What you need to do for this test is:
> 
> 1. Create a `pytest.fixture`for the hmi test file much like `aia171_test_map` at the top of the file. Here is the [hmi test file to see how to access the test file](https://github.com/sunpy/sunpy/blob/cd8618e961b4140f2d65cc955f48760098e068f9/sunpy/map/sources/tests/test_hmi_source.py).
> 2. You will will want to assert that the `hmi_map.heliographic_longitude` is the same as if you opened the hmi file, loaded the header/meta information and done the operation on the crln_obs yourself.

I have run the tests locally and it passed, hope it's correct.